src/scripts/settings/core.lua

```diff
@@ -1,5 +1,74 @@
 -- Paketversion für Thirdparty Skripte speichern
 MG_Version = getPackageInfo("MorgenGrauen", "version")
 
+-- Gedöns... weil Tippfaul
+local MG_DBName = "MG_Settings"
```

Comment:
Am besten ebenfalls in Table sammeln wie MG.DBName oder MG.Core.DBName

Reply:
MG_Version war eigentlich für Thirdparty Skripte gedacht.

a) um zu prüfen ob unser Paket installiert ist und 
b) gleich auf eine bestimmte Version(shöhe) testen zu können

Letzteres hatte ich noch nicht umgesetzt und am Ende habe ich der Einfachheit halber beim Beispiel für die Thirdpartynutzung auf das Vorhandensein von ```getConfig``` getestet^^

Generell wäre es aber schon noch sinnvoll allen globalen Kram aus unserem Paket in eine MG Tabelle zu packen. Die kann dann auch ruhig in jeder Session neu initialisiert werden, ist ja nix drin was eine Session überdauern muss. Muss dann aber in einem Event geschehen das beim Laden der Skripte 1x ausgeführt wird, nicht nur bei der Installation. Bin mir sicher da gab es eins :)